crates/datagen/src/lib.rs

```diff
@@ -245,6 +301,31 @@ fn parse_range_for_datetime(
     }
 }
 
+/// This is our max precision for decimals.
+const FELDERA_MAX_DECIMAL_PRECISION: u32 = 28;
+
+/// THis is our max scale for decimals.
```

Comment:
typo?
Our definitions in Java are:
```
    public static final int MAX_PRECISION = 28;   // Total digits. This limit comes from the Rust decimal library
    public static final int MAX_SCALE = 10;      
```

Frankly, I don't know why the scale needs to be limited to 10.